pymemcache/client/base.py

```diff
@@ -304,6 +305,9 @@ def set(self, key, value, expire=0, noreply=None):
                   from the cache, or zero for no expiry (the default).
           noreply: optional bool, True to not wait for the reply (defaults to
                    self.default_noreply).
+          flag: remcached operation flag
```

Comment:
The `flag` is not specific to remcached, but rather [one component](https://github.com/memcached/memcached/blob/master/doc/protocol.txt#L149) of the memcached protocol's underlying storage command API. Let's update the documentation to reflect this.